Dockerfile

```diff
@@ -1,16 +1,11 @@
-FROM concourse/buildroot:git
-
-RUN \
-  cd /usr/bin/ && \
-  curl \
-    -L \
-    -O \
-    https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 && \
-  mv jq-linux64 jq && \
-  chmod +x jq
-
-ADD cmd/check/check /opt/resource/check
-ADD cmd/in/in /opt/resource/in
-ADD cmd/out/out /opt/resource/out
+FROM golang:alpine as builder
```

Comment:
Can we find a way to add the necessary utilities without changing the `FROM`?

It's hard to test the impact of changing this. I'd assume that the size of the resultant image will increase, as `concourse/buildroot` is very small.

Reply:
The image is actually smaller that concourse/buildroot but I understand the point.  I wasn't how  concourse/buildroot was created so wasn't sure how to add the needed utilities.  I can look into reverting to use concourse/buildroot.

Reply:
Reverted back to concourse:buildroot